movo_robot/movo_bringup/launch/manipulation/moveit_action_servers.launch

```diff
@@ -0,0 +1,43 @@
+<launch>
```

Comment:
i think this should be in nightingale_manipulation

Reply:
If this change is critical to get the arms to work on the MOVO I think it should stay here since others will need it to use the arms reliably. From what I understand I thought this is necessary to get the arms to work and should now be shipped with the movo so it can work regardless of what we are running in nightingale